### DISCUSSION TITLE
XState 5

### DISCUSSION THREAD

**Discussion by robertsargent** [koordinates/xstate-tree#52] [General]
Hi @UberMouse, great work. I've been thinking about how to implement something like this for a new project I'm working on. Do you plan to support XState 5 in the near future?

**Comment by UberMouse** [comment]
Indeed, I have recently started work on a v5 update which will support xstate v5. Not sure on an ETA but I'm hoping not too much longer!

**Comment by UberMouse** [comment]
@robertsargent there's support for xstate v5 released under the @next tag, or version 5.1.0-next.1 (had some snafus setting up semantic-release for next prereleases eating up versions :sweat_smile: 

Usual caveats around prerelease versions, hasn't had extensive testing and I haven't updated any documentation (not that much has changed)

**Reply by robertsargent** [reply]
Great, thanks @UberMouse! I'll give it a whirl 🙂
